### PR TITLE
Update Order Page View.md

### PR DESCRIPTION
**Pull request by MatShir**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | Currently, the product list displays the product by the order in which the products are added to the cart. So the first page of the product list into the order page is the first one to be added and the last page on the product list displays the latest products added. <br /><br />A product added through the BO's order page was displayed on the page where the product was added and then displayed at the end after a refresh. Now, with the new refresh system of the product list, the user won't know if the product is successfully added since the product will be added directly on the last page. <br /><br />So, to bring more awareness to the user, when a product is added, the page will display a success alert. The product list will now display the product by the order anti chronological, the latest product added to the cart are the first one to be displayed.
| Fixed ticket? | https://github.com/PrestaShop/PrestaShop/issues/21500

### CODE REVIEW

**Comment by marionf** [comment]
Hey @MatShir Could you resolve the conflicts before the review please ?

**Comment by MatShir** [comment]
Conflict resolved! Can I merge it? @marionf

File: back-office/orders/orders/Order Page View.md [hunk 1]
```diff
@@ -242,26 +242,26 @@ If the customer is a guest:
...
+The button to transform a guest into a customer is removed. The feature is still available on the customer page. The order is not the place to handle customer management. It was removed to not encourage the merchant to transform Guest to a Customer account by letting a button on the order page.
 
 3. **Shipping and Invoice address**
 
 Each address has **a button** to update or change it. The button opens a list to select to edit or update it.
-Editing an address actually creates a new address and soft-deletes the old one. So, if the edited address is used by an old order, it will not be updated with the new address.
+Editing an address creates a new address and soft-deletes the old one. So, if the edited address is used by an old order, it will not be updated with the new address.
 
-[Edit/Change adresse behaviors](https://invis.io/YKTGGFZAMCB#/386397109_Edit_Adress)
+[Edit/Change Adresse behaviors](https://invis.io/YKTGGFZAMCB#/386397109_Edit_Adress)
```
Comment (marionf) [inline back-office/orders/orders/Order Page View.md:252]:
```suggestion
[Edit/Change Addresse behaviors](https://invis.io/YKTGGFZAMCB#/386397109_Edit_Adress)
```

File: back-office/orders/orders/Order Page View.md [hunk 2]
```diff
@@ -374,24 +380,29 @@ If the discount is updated the order remains the same.
...
 
 3. **Remove products from the order**
 
-Deleting one or several products is very easy : a remove icon is available to remove one or several quantities of a paid product. Current bug : if I delete the paid product, it will also delete the gifted product (the voucher is still displayed). The deletion should only delete the paid product.
+Deleting one or several products is very easy: a remove icon is available to remove one or several quantities of a paid product. 
+
+When a product is removed from the order, a notification is displayed: "The item was successfully removed." in Admin.Notifications.Error.
+Then the product list is refreshed behind the loading spinner to check if any gift product has to be removed as well. The user is redirected to the page where the user removed the product. If the removed product was on the last page, the user is redirected to the second-last page.
```
Comment (marionf) [inline back-office/orders/orders/Order Page View.md:403]:
```suggestion
Then the product list is refreshed behind the loading spinner to check if any gift product has to be removed as well. The user is redirected to the page where he/she was when he/she removed the product. If the removed product was on the last page, the user is redirected to the second-last page.
```

File: back-office/orders/orders/Order Page View.md [hunk 3]
```diff
@@ -488,11 +499,11 @@ The table **list of the latest order status** is displayed composed:
 
 Below the list, a drop list of all the status available is displayed to update the current status through the **button Update Status** next to it. Updating the order status will trigger all the actions set by the order status.
 
-On the bottom of the bock, there is the **Order comment**. The order comment saves a comment for the corresponding order. It is closed by default when it is empty and open when filled. To add a comment when it is empty, the user has to click on the + button and has to press on the save button to save it.
+On the bottom of the bock, there is the **Order comment**. The order comment saves a comment for the corresponding order. It is closed by default when it is empty and open when filled. To add a comment when it is empty, the user has to click on the + button and has to press the save button to save it.
```
Comment (marionf) [inline back-office/orders/orders/Order Page View.md:502]:
```suggestion
On the bottom of the block, there is the **Order comment**. The order comment saves a comment for the corresponding order. It is closed by default when it is empty and open when filled. To add a comment when it is empty, the user has to click on the + button and has to press the save button to save it.
```

File: back-office/orders/orders/Order Page View.md [hunk 4]
```diff
@@ -505,44 +516,44 @@ Each table’s rows as 2 possible **button actions**:
...
 
-After adding, the user can display **all the details of the transaction by pressing the Detail button**. It will show below the arrow **the number, brand, expiration date and owner of the credit card** for any payment module.
+After adding, the user can display **all the details of the transaction by pressing the Detail button**. It will show below the arrow **the number, brand, expiration date, and owner of the credit card** for any payment module.
 
 **If the order’s status** is considered **as paid then before adding a transaction** the user has to specify in **which generated invoices from a droplist** it will be added.
 
 When the status accepted (waiting for payment) you can change the currency of the order with a drop list.
 
-If the shop has different currencies, a droplist of shop's currencies is displayed to change the order currency. The change rate is based on the latest conversion rate save by the shop.
+If the shop has different currencies, a droplist of the shop's currencies is displayed to change the order currency. The change rate is based on the latest conversion rate save by the shop.
```
Comment (marionf) [inline back-office/orders/orders/Order Page View.md:548]:
```suggestion
If the shop has different currencies, a droplist of the shop's currencies is displayed to change the order currency. The change rate is based on the latest conversion rate saved in the shop.
```

**Review by marionf — CHANGES_REQUESTED** [review]
Just some suggestions

**Comment by MatShir** [comment]
Thanks @marionf

**Pull request merged**